demo/asset_card.js

```diff
@@ -307,7 +307,7 @@ shakaDemo.AssetCard = class {
   addStoreButton() {
     /**
      * Makes the contents of the button into an MDL icon, and moves it into the
-     * upper-righthand corner with CSS styles.
+     * upper-right hand corner with CSS styles.
```

Comment:
My American English sensibilities make me want to change this to "upper-right-hand".  But for the record, automation should sometimes take precedence over vibes.  If cspell disagrees with me, go with cspell.

Reply:
CSpell only marked `righthand` as a non existent word, fix was done by myself. Fixed then!
Unfortunately this tool very rarely suggests proper spelling. Hope it will be improved in the future.

Reply:
Seems like dictionaries/style manuals have upper-right-hand when it's used as an adjective and upper-right hand when used as a noun.